esmvalcore/preprocessor/_io.py

```diff
@@ -103,7 +103,8 @@ def concatenate(cubes):
     raise ValueError('Can not concatenate cubes.')
 
 
-def save(cubes, filename, optimize_access='', compress=False, **kwargs):
+def save(cubes, filename, optimize_access='',
+         compress=False, dryrun=False, **kwargs):
```

Comment:
@valeriupredoi The `dryrun` option is not used in this function nor mentioned in the docstring.

Reply:
indeed, good catch - Ill have to add documentation for it! I need the arg there because it's passed by `_recipe.py` and used in the `PreprocessorFile` class from `__init__`